project.clj

```diff
@@ -2,10 +2,13 @@
   :description "A Clojure library with higher level CSV parsing functionality"
   :url "http://github.com/metasoarous/semantic-csv"
   :license {:name "Eclipse Public License"
-            :url "http://www.eclipse.org/legal/epl-v10.html"}
+            :url  "http://www.eclipse.org/legal/epl-v10.html"}
```

Comment:
Oh... I see, you were going for alignment. I usually eschew this cause it's more work to update, but I'm happy to leave it be :-)